src/utils/index.ts

```diff
@@ -1,7 +1,10 @@
-export { getKeycloakRoles } from './getKeycloakRoles'
-export { axios } from './axios'
-export { initLdClient, updateLdUser, getFeatureFlag } from './feature-flags'
-export { fetchConfig } from './fetchConfig'
-export { isSame } from './isSame'
-export { navigate } from './navigate'
-export { sleep } from './sleep'
+export * from './AxiosInstance'
```

Comment:
I wish there was a way around this somehow..

Reply:
We don't have to use the index file, but it can make for cleaner imports.

I prefer this over "import default as X from ..", lol.